schemas/ingest-metrics.v1.schema.json

```diff
@@ -22,32 +22,33 @@
         "type": {
           "description": "The metric type. [c]ounter, [d]istribution, [s]et. Relay additionally defines Gauge, but that metric type is completely unsupported downstream.",
           "type": "string",
-          "pattern": "^[cds]$"
+          "enum": ["c", "d", "s"]
         },
         "timestamp": {
           "description": "The timestamp at which this metric was being sent. Relay will round this down to the next 10-second interval.",
           "type": "integer"
         },
         "tags": {
-          "$ref": "#/definitions/StringToString"
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
         },
         "value": {
           "anyOf": [
             {
-              "type": "integer"
+              "title": "number_metric_value",
```

Comment:
Does that also work in python?

Reply:
I believe it will produce `NumberMetricValue = int`, but i havent' checked

Reply:
I didn't see it produce anything but TypedDict's before but haven't checked recently

Reply:
I checked, and it appears it produces:

```
[...]
    value: Required[Union["NumberMetricValue", "NumbersMetricValue"]]
    """Required property"""



NumberMetricValue = Union[int, float]
"""number_metric_value."""



NumbersMetricValue = List[Union[int, float]]
"""numbers_metric_value."""
```